### DISCUSSION TITLE
Dropdown panel height

### DISCUSSION THREAD

**Discussion by gallwood** [BlazorFluentUI/BlazorFluentUI#351] [Q&A] [answered]
Hi,

Is it possible to restrict the Panel height of the Dropdown? I have a collection of 50 items, so when the dropdown shows the list it is actually the full height of the screen - doesn't look that great. I wanted to a maximum height if possible.

Thanks

Graham

**Comment by vnbaaij** [comment, marked as answer]
A parameter `DropdownHeight` has been added to the Dropdown component for this in V5.5.0

**Reply by limefrogyank** [reply]
That seems to have broken dropdown a bit.  See the demo:  https://www.blazorfluentui.net/dropdownPage

Without the additions, you could probably also change the height using css:
```
.ms-Dropdown-itemsWrapper {
   max-height:300px;
}
```

**Reply by vnbaaij** [reply]
Yeah, I made a booboo with the DropdownHeight by giving it the height of the actual dropdown box . Fixed now. It is initilized at 0 and then takes the origina calculated height otherwise it takes the height given.

**Reply by gallwood** [reply]
Thanks guys.

**Reply by gallwood** [reply]
Perfect @vnbaaij , just tried this out and it works a treat. Thank you.
